bodhi/server/models.py

```diff
@@ -1631,10 +1627,8 @@ class Update(Base):
 
     __tablename__ = 'updates'
     __exclude_columns__ = ('id', 'user_id', 'release_id')
-    __include_extras__ = ('meets_testing_requirements', 'url',)
-    __get_by__ = ('title', 'alias')
```

Comment:
Removing `title` from this container will block access to updates via `/updates/{title}`(like: bodhi.fedoraproject.org/updates/phpdoc-2.9.0-7.fc29)  so maybe we should add note in release notes in B/I section?

Reply:
Added!